app/javascript/packs/selectors/answersSelector.js

```diff
@@ -1,3 +1,19 @@
 export function getAnswerForQuestionAndBuilding(questionId, buildingId, state) {
   return state.buildings[buildingId].answers[questionId]
 }
+
+//##returns (from the questions provided) how many are remaining to answer
+//##if no buildingId is provided, we are not viewing a specific building
+// so we should not render how many questions are remaining for a building
+//## if no questions are provided, the building did not have any questions for the users so they have no questions
+//to answer for that building
+export function getRemainingAnswersforCategory(questions, buildingId, state) {
```

Comment:
I mentioned this earlier but I think that this type of extensive logic is a bit beyond what selectors should be for; maybe break this out into a separate importable function that uses parameters that are more easily selectable

Reply:
@HacksForHugs would you recommend a `isUnanswered` filter function inside of `utils`? Not sure what much else could be broken out here, but I can see the usefulness of a shared `isUnanswered` function for `Answer` objects.